### PR TITLE
fix: Log error on table resolver exception/error

### PR DESCRIPTION
**Pull request by yevgenypats**
The previous logger was raising an exception inside an exception handler

### CODE REVIEW

File: cloudquery/sdk/scheduler/scheduler.py [hunk 1]
```diff
@@ -93,14 +93,15 @@ def resolve_table(
         try:
             if depth == 0:
                 self._logger.info(
-                    "table resolver started", table=resolver.table.name, depth=depth
+                    "table resolver started what", table=resolver.table.name, depth=depth
```
Comment (hermanschaaf) [inline cloudquery/sdk/scheduler/scheduler.py:96]:
```suggestion
                    "table resolver started", table=resolver.table.name, depth=depth
```

File: cloudquery/sdk/scheduler/scheduler.py [hunk 2]
```diff
@@ -93,14 +93,15 @@ def resolve_table(
...
                 )
             else:
                 self._logger.debug(
                     "table resolver started", table=resolver.table.name, depth=depth
                 )
             total_resources = 0
             for item in resolver.resolve(client, parent_item):
+                self._logger.info("resolving resource")
```
Comment (hermanschaaf) [inline cloudquery/sdk/scheduler/scheduler.py:104]:
```suggestion
```

File: cloudquery/sdk/scheduler/scheduler.py [hunk 3]
```diff
@@ -113,6 +114,7 @@ def resolve_table(
                         exception=e,
                     )
                     continue
+                self._logger.info("resolved resource")
```
Comment (hermanschaaf) [inline cloudquery/sdk/scheduler/scheduler.py:117]:
```suggestion
```

File: cloudquery/sdk/scheduler/scheduler.py [hunk 4]
```diff
@@ -125,25 +127,26 @@ def resolve_table(
                     )
                     table_resolvers_started += 1
                 total_resources += 1
+            self._logger.info("what end")
```
Comment (hermanschaaf) [inline cloudquery/sdk/scheduler/scheduler.py:130]:
```suggestion
```

File: cloudquery/sdk/scheduler/scheduler.py (hunk 1 above)
Comment (yevgenypats) [reply cloudquery/sdk/scheduler/scheduler.py:96]: yeah forgot it. removed.

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**